src/redisearch_rs/trie_bencher/build.rs

```diff
@@ -6,20 +6,25 @@ fn main() {
     let root = git_root();
 
     // Construct the correct folder path based on OS and architecture
+    let target_os = env::var("CARGO_CFG_TARGET_OS").unwrap();
     let lib_dir = {
         let target_arch = env::var("CARGO_CFG_TARGET_ARCH").unwrap();
         let target_arch = match target_arch.as_str() {
             "aarch64" => "arm64v8",
             "x86_64" => "x64",
             _ => &target_arch,
         };
-        let target_os = env::var("CARGO_CFG_TARGET_OS").unwrap();
+
         root.join(format!(
             "bin/{target_os}-{target_arch}-release/search-community/deps/triemap"
         ))
     };
 
     assert!(std::fs::exists(lib_dir.join("libtrie.a")).unwrap());
+    // ensure that undefined symbols are allowed - that's seems to be the case on macOS but not un Ubuntu in CI.
```

Comment:
Let's elaborate in the comment as to why we have undefined symbols and that's OK here.